### PR TITLE
Mobile footer

### PR DESCRIPTION
**Pull request by nhsz**
Add Footer for mobile version

- Adds `Footer` component to `Layout`.
- Adds `/applicants` & `/about` page placeholders.
- Adds EF, Twitter & GitHub SVG logos.

PREVIEW: https://deploy-preview-119--thirsty-meitner-cae269.netlify.app/#footer

### CODE REVIEW

File: src/components/UI/common/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+import Image from 'next/image';
+import { FC } from 'react';
+import efLogoSVG from '../../../public/images/ef-logo.svg';
+import githubLogoSVG from '../../../public/images/github-logo.svg';
+import twitterLogoSVG from '../../../public/images/twitter-logo.svg';
+import { ImportantText, PageText } from '../text';
+
+export const Footer: FC = () => {
+  return (
+    <Flex id='footer' bgGradient='linear(to-b, #E07861 10%, #E06361 100%)' d='column'>
```
Comment (samajammin) [inline src/components/UI/common/Footer.tsx:11]: Should we use variables for these hex colors?

File: src/components/UI/common/Footer.tsx [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+      <Center px={5} py={8}>
+        <Image src={efLogoSVG} alt='Ethereum Foundation logo' height={64} width={200} />
+      </Center>
+
+      <Center px={5}>
+        <Divider borderColor='#e5e5e5' opacity={1} />
```
Comment (samajammin) [inline src/components/UI/common/Footer.tsx:17]: Same here - should this color be a variable?

**Review by samajammin — APPROVED** [review]
Couple comments but looks good!

File: src/components/UI/common/Footer.tsx [hunk 3]
```diff
@@ -0,0 +1,105 @@
+import { Box, Center, Divider, Flex, Link, Stack } from '@chakra-ui/react';
+import Image from 'next/image';
+import { FC } from 'react';
+import efLogoSVG from '../../../public/images/ef-logo.svg';
```
Comment (samajammin) [inline src/components/UI/common/Footer.tsx:4]: Minor nitpick but I do like to see consistent order of imports, e.g. something like:
1. libraries (react, chakra-ui)
2. components
3. images

Nothing blocking this PR but something to consider moving forward.

Comment (samajammin) [reply src/components/UI/common/Footer.tsx:4]: You may have already thought this through :) LMK if you have an opinion.

File: src/components/UI/common/Footer.tsx (hunk 2 above)
Comment (nhsz) [reply src/components/UI/common/Footer.tsx:17]: will move the colors to variables 👌

File: src/components/UI/common/Footer.tsx (hunk 3 above)
Comment (nhsz) [reply src/components/UI/common/Footer.tsx:4]: I'm currently using the [`organizeImports` ](https://eshlox.net/2019/12/02/vscode-automatically-organize-typescript-imports/) from vscode which sorts them alphabetically. I'll check if I can add an eslint rule to enforce that pattern 👍.

Comment (samajammin) [reply src/components/UI/common/Footer.tsx:4]: Oh, nice! Didn't know about that - I like it 👍

File: src/components/UI/common/Footer.tsx (hunk 1 above)
Comment (nhsz) [reply src/components/UI/common/Footer.tsx]: Moved to `theme`

**Pull request merged**